configure.ac

```diff
@@ -561,7 +561,7 @@ case "${host}" in
 	dnl syscall(2) and secure_getenv(3) are exposed by _GNU_SOURCE.
 	JE_APPEND_VS(CPPFLAGS, -D_GNU_SOURCE)
 	abi="elf"
-	AC_DEFINE([JEMALLOC_PURGE_MADVISE_DONTNEED_ZEROS])
+	AC_DEFINE([JEMALLOC_PURGE_MADVISE_DONTNEED_ZEROS], [ ])
```

Comment:
What's the difference between these?

Reply:
The default adds a value 1 to the macro (i.e. it does #define JEMALLOC_PURGE_MADVISE_DONTNEED_ZEROS 1) , which we don't use.